template/script.sh.erb

```diff
@@ -14,16 +12,7 @@ elsif working_dir.file?
 end
 %>
 
-# Setup environment.
-APP_ROOT="<%= app_root.to_s %>"
-CODE_SERVER_DATAROOT="$HOME/.local/share/bc_osc_codeserver"
-
-# Path to Code Server binary. /code-server/ is expected to be a symlink to #{CODE_SERVER_FOLDER}/bin/code-server
-#
-# ln -s code-server-3.3.1-linux-x86_64 code-server
-CODE_SERVER="${CODE_SERVER:-${APP_ROOT}/code-server/bin/code-server}"
-
-# Ensure a dataroot for Code Server.
+CODE_SERVER_DATAROOT="$HOME/.local/share/code-server"
```

Comment:
I think using `code-server` is a safe choice given folks would want to have the same experience no matter how they get there (through VDI, X11 forwarding, remote execution or this app). But it's worth pointing out now, just because it would be painful to change later.

Reply:
I agree that `code-server` is a safe choice.